src/views/postUpdation/TextPostUpdation.tsx

```diff
@@ -45,6 +49,9 @@ export const TextPostUpdation: React.FC<ITextPostUpdationProps> = ({
     post.origins,
   );
   const [htmlContent, setHtmlContent] = useState(post.content);
+  const [previewImageUrl, setPreviewImageUrl] = useState<string>(
+    post.previewImageUrl as string,
```

Comment:
What if it's undefined? Maybe better to do like this
```suggestion
    post.previewImageUrl ?? '',
```

Reply:
Fixed